recipes/debug_assert/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, tools
+import os
+
+class DebugAssert(ConanFile):
+    name = 'debug_assert'
+    description = 'Simple, flexible and modular assertion macro'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'http://foonathan.net/blog/2016/09/16/assertions.html'
+    license = 'zlib'
```

Comment:
```suggestion
    license = 'Zlib'
```

https://spdx.org/licenses/Zlib.html